### ISSUE TITLE
I've found a bug about the commitment loss  in `taming/modules/vqvae/quantize.py`

### ISSUE THREAD

**Issue by wcshin-git** [CompVis/taming-transformers#57]
Hi @pesser and @rromb,
thanks for the amazing work!

I read your paper and code line-by-line, and I found that the commitment loss weighting factor (beta) in the code is slightly different from what is mentioned in the paper.

<img width="488" alt="fig1" src="https://user-images.githubusercontent.com/76698585/121781797-5f301980-cbe1-11eb-80b8-b76ed1890aa3.png">
<img width="998" alt="fig2" src="https://user-images.githubusercontent.com/76698585/121781803-6a834500-cbe1-11eb-951e-d6181585597e.png">
As you mentioned in your paper, you used beta=0.25 and this should be multiplied by commission loss(I think you used beta=0.25 as mentioned in VQ-VAE(A Oord et al., 2017)).

But in your code, beta is multiplied by the second term in Eq (4) of the paper.

If I'm right, I'd appreciate it if you merge this [PR](https://github.com/CompVis/taming-transformers/pull/58). Because I want to be a contributor to your amazing work:)
BEFORE:
```
# at 69-th line in `taming/modules/vqvae/quantize.py`
# self.beta = 0.25(It is fixed at 0.25 in your code)    
loss = torch.mean((z_q.detach()-z)**2) + self.beta * \           
 torch.mean((z_q - z.detach()) ** 2)
```

AFTER:
```
loss = self.beta * torch.mean((z_q.detach()-z)**2) +  \           
 torch.mean((z_q - z.detach()) ** 2)
```

Thank you!

**Comment by pesser** [comment]
Closed by #58 . Set `legacy=False` in the quantizer to use it. Thanks!
